web/src/routes.js

```diff
@@ -20,6 +21,7 @@ const routes = [
     nonPrivate: false
   },
   { path: '/admin/edit-account', component: EditAccount, nonPrivate: false },
+  { path: '/admin', component: AdminDashboard, nonPrivate: false },
```

Comment:
I don't fully understand this, but when I put the `/admin` path preceding `/admin/*` paths, the former overwrote the latter. This is probably because I was expecting it to work like Django views. 😄

Reply:
React-router stops on the first path object that matches the page location, based on the order the path objects are inserted.  It's also basically a `startsWith` match, so it can be aggressive.  There are several workarounds - putting the more general paths later in the insertion order is one.  Another is to add an `exact` prop which tells the router to only match exactly, not generally.  I think I'd recommend we add the `exact` prop anyway, to avoid any weirdness if we add more pages in the future.

```suggestion
  { path: '/admin', component: AdminDashboard, nonPrivate: false, exact: true },
```

(The `nonPrivate` property is pulled from these objects, and all other properties are passed on to a react-router [`<Route>` component](https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/Route.md).)